content/src/main/content/jcr_root/etc/designs/cqsm/clientlibs/css/progressBar.css

```diff
@@ -1,3 +1,22 @@
+/*-

```

Comment:
btw, is that css still in use? I think we've removed progress bar long time ago?